src/Diagrams/Swimunit/Dotmatrix.hs

```diff
@@ -0,0 +1,843 @@
+{-# LANGUAGE NoMonomorphismRestriction #-}
+
+module Diagrams.Swimunit.Dotmatrix
+where
+
+import qualified Data.Map as Map
+
+import Diagrams.Prelude
+import Diagrams.Backend.SVG.CmdLine
```

Comment:
Hi Chris,

cool. I'll give it a try and you will see either a new checkin or a 
desperate mail.

Regards
     Wolfram

On 18.09.2015 21:27, Chris wrote:
>
> In src/Diagrams/Swimunit/Dotmatrix.hs 
> <https://github.com/diagrams/diagrams-contrib/pull/55#discussion_r39892052>:
>
> > @@ -0,0 +1,843 @@
> > +{-# LANGUAGE NoMonomorphismRestriction #-}
> > +
> > +module Diagrams.Swimunit.Dotmatrix
> > +where
> > +
> > +import qualified Data.Map as Map
> > +
> > +import Diagrams.Prelude
> > +import Diagrams.Backend.SVG.CmdLine
>
> That's fine. I'd say replace all the |Diagram B| with |QDiagram b V2 n 
> Any| and |ghc| should tell you what to do if there's any errors. If 
> you're still getting stuck you can ask here or on irc.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/diagrams/diagrams-contrib/pull/55/files#r39892052>.
>